db/migrate/20211215120415_create_email_configurations.rb

```diff
@@ -0,0 +1,31 @@
+class CreateEmailConfigurations < ActiveRecord::Migration[5.0]
+  def change
+
+    # Use case insensitive collation for emails although by spec local part is sensitive.
+    # For practical purposes we don't want to support email with only case differences.
+    # Oracle and Postgres don't have case insensitive collations, we will rely on app logic for it.
+    collation_ci = case
+                 # when System::Database.postgres?
+                 #   supported only with version 13+
+                 #   { collation: "und-x-icu" }
+                 when System::Database.mysql?
+                   { collation: "utf8_general_ci" }
+                 else
+                   {}
+                 end
```

Comment:
I think this will use the config/database.yml options by default. Also no need to add it for every column.
I say this is not needed

Reply:
It is to add `_ci` collation for columns that should be case insensitive. Unfortunately seems to work only on MySQL. Some fields are better off with `_bin` collation some with `_ci`. It mostly matters for unique indices. Otherwise we need to ensure uniqueness on the app level which is not so reliable.

Do you want no `_ci` indices?